oci/private/image_index.sh.tpl

```diff
@@ -24,7 +24,7 @@ function add_image() {
         local platform=$("${JQ}" -c '{"os": .os, "architecture": .architecture, "variant": .variant, "os.version": .["os.version"], "os.features": .["os.features"]} | with_entries(select( .value != null ))' "${image_path}/blobs/${config_blob_path}")
         "${JQ}" --argjson platform "${platform}" \
                 --argjson manifest "${manifest}" \
-                '.manifests |= [$manifest + {"platform": $platform}]'\
+                '.manifests |= . + [$manifest + {"platform": $platform}]'\
```

Comment:
I do like @tobiaskohlbau's use of `+=` in #637. Which do you prefer here?

Reply:
+= probably better, let's do that